src/software/python_bindings.cpp

```diff
@@ -207,6 +209,14 @@ PYBIND11_MODULE(python_bindings, m)
         .def("radius", &Circle::radius)
         .def("area", &Circle::area);
 
+    py::class_<RobotConstants>(m, "RobotConstants")
+        .def_readwrite("max_force_dribbler_speed_rpm",
+                       &RobotConstants::max_force_dribbler_speed_rpm)
+        .def_readwrite("robot_max_speed_m_per_s",
+                       &RobotConstants::robot_max_speed_m_per_s)
+        .def_readwrite("robot_max_ang_speed_rad_per_s",
+                       &RobotConstants::robot_max_ang_speed_rad_per_s);
```

Comment:
The rest of fields in `RobotConstants` should also be added here so they are available in case we need them in the future.